### PR TITLE
Ticket 13: Create the common UI

### PR DESCRIPTION
**Pull request by HuynhKane**
### Description
- Create 6 composable
- Have NOT done the onClick of MiniButton and ProductCard
### Include
- Add 6 icons svg
- UI Composables

| :---:   | :---: | 
| DialogView |
image
|
| ItemFunction |
image
|
| Header Of Screen |
image
|
| Functions Container |Admin: 
image
User:
image

|
| MiniButton |
image
|
| ProductCard |
image
image

|

### CODE REVIEW

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/HeaderOfScreen.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        if (!isMainScreen) {
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.Start,
+                verticalAlignment = Alignment.CenterVertically
+
+            )
+            {
+                Image(painter = painterResource(id = R.drawable.icons8_back),
```
Comment (PhunnMoc) [inline app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/HeaderOfScreen.kt:50]: It's not just Interface IU !! This button can be clicked and do Sth

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ItemFunction.kt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.warehousemanagement.R
+
+
+
+@Composable
+fun ItemFunction(functionName: String, iconResource: Int, size: Dp) {
```
Comment (PhunnMoc) [inline app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ItemFunction.kt:27]: What happen when I click into this Itemfunction? Nothing !!

**Review by PhunnMoc — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/DialogView.kt [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import com.example.warehousemanagement.R
+
+// Composable function for a custom dialog view
```
[the remark this answers is not in the document]
Comment (HuynhKane) [reply app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/DialogView.kt:26]: Done

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/FunctionContainer.kt [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                repeat(3) {
+                    ItemFunction("Function", R.drawable.icons8_square_function,100.dp)
+                }
+            }
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceEvenly
+            ) {
+
+                repeat(3) {
```
[the remark this answers is not in the document]
Comment (HuynhKane) [reply app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/FunctionContainer.kt]: done

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/HeaderOfScreen.kt (hunk 1 above)
Comment (HuynhKane) [reply app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/HeaderOfScreen.kt]: done

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ItemFunction.kt (hunk 2 above)
Comment (HuynhKane) [reply app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ItemFunction.kt]: done

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/MiniButton.kt [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.warehousemanagement.R
+
+@Composable
+fun MiniButton(onClick: () -> Unit) {
```
[the remark this answers is not in the document]
Comment (HuynhKane) [reply app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/MiniButton.kt]: done

File: app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ProductCard.kt [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+                fontSize = 12.sp)
+        }
+    }
+}
+
+
+@Preview(showBackground = true)
+@Composable
+fun PreviewProductCard() {
+    ProductCard(
```
[the remark this answers is not in the document]
Comment (HuynhKane) [reply app/src/main/java/com/example/warehousemanagement/domain/feature/ui/common/ProductCard.kt:124]: done

File: app/src/main/res/drawable/icons8_squared_menu_80.xml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    android:width="80dp"
+    android:height="80dp"
+    android:viewportWidth="80"
+    android:viewportHeight="80">
+  <path
+      android:pathData="M59,53.867C59.804,53.866 60.609,53.865 61.438,53.863C62.283,53.867 63.129,53.871 64,53.875C65.268,53.869 65.268,53.869 66.563,53.863C67.367,53.865 68.171,53.866 69,53.867C69.743,53.868 70.485,53.869 71.25,53.871C73,54 73,54 74,55C74.098,56.33 74.131,57.666 74.133,59C74.135,60.207 74.135,60.207 74.137,61.438C74.133,62.283 74.129,63.129 74.125,64C74.131,65.268 74.131,65.268 74.137,66.563C74.135,67.367 74.134,68.171 74.133,69C74.132,69.743 74.131,70.485 74.129,71.25C74,73 74,73 73,74C71.67,74.098 70.334,74.131 69,74.133C68.196,74.134 67.391,74.135 66.563,74.137C65.717,74.133 64.871,74.129 64,74.125C63.154,74.129 62.309,74.133 61.438,74.137C60.633,74.135 59.829,74.134 59,74.133C58.257,74.132 57.515,74.131 56.75,74.129C55,74 55,74 54,73C53.902,71.67 53.869,70.334 53.867,69C53.866,68.196 53.865,67.391 53.863,66.563C53.867,65.717 53.871,64.871 53.875,64C53.871,63.154 53.867,62.309 53.863,61.438C53.865,60.633 53.866,59.829 53.867,59C53.868,58.257 53.869,57.515 53.871,56.75C54.123,53.337 56.053,53.872 59,53.867Z"
```
[the remark this answers is not in the document]
Comment (HuynhKane) [reply app/src/main/res/drawable/icons8_squared_menu_80.xml:7]: done

**Review by PhunnMoc — APPROVED** [review]

**Pull request merged**